src/operators.jl

```diff
@@ -115,9 +115,19 @@ function union{T<:AbstractGraph}(g::T, h::T)
     return r
 end
 
+isunique(iter::Range) = true
+isunique(iter::Set) = true
+isunique(iter) = (length(unique(iter)) == length(iter))
+
 #@doc "filter g to include only the vertices present in iter which should not have duplicates
 #returns the subgraph of g induced by set(iter) along with the mapping from the old vertex names to the new vertex names" ->
-function inducedsubgraph{T<:AbstractGraph}(g::T, iter)
+function induced_subgraph{T<:AbstractGraph}(g::T, iter)
+    !isunique(iter) && error("Vertices in subgraph list must be unique")
```

Comment:
`unique` may preserve order now, but I'm not comfortable assuming it's guaranteed to in the future (based on past behavior where they're `collect()`ing a set).

Is trying to get an induced subgraph on a nonunique collection a common occurrence? Why?